### PR TITLE
Rename develop branch to main

### PR DESCRIPTION
**Pull request by amirhosv**
*Issue #, if available:*
CryptoAlg-1107

*Description of changes:*
Reflecting the renaming of develop to main in our code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alexw91** [comment]
Can we name the branch `main` instead of `mainline`? That'll align ourselves with [s2n-tls](https://github.com/aws/s2n-tls), [s2n-quic](https://github.com/aws/s2n-quic), and [aws-lc](https://github.com/aws/aws-lc) who all use the `main` branch as the primary one used for development.

**Comment by amirhosv** [comment]
> Can we name the branch `main` instead of `mainline`? That'll align ourselves with [s2n-tls](https://github.com/aws/s2n-tls), [s2n-quic](https://github.com/aws/s2n-quic), and [aws-lc](https://github.com/aws/aws-lc) who all use the `main` branch as the primary one used for development.

done

File: README.md [hunk 1]
```diff
@@ -6,21 +6,6 @@ As of 2.0.0, algorithms exposed by ACCP are primarily backed by [AWS-LC](https:/
...
-Both of these tests are flakey because they include entropy generation tests
-(specificaly, the [Dieharder tests](http://webhome.phy.duke.edu/~rgb/General/dieharder.php)).
-Entropy tests are unavoidably flakey because there is always a possibility that a high-quality
-random number generator will output data which looks non-random.
-(For example, even a fair coin will come up heads ten times in a row about one in a thousand trials.)
-
-| Build Name | master branch | develop branch |
-| ---------- | ------------- | -------------- |
-| x86        | ![Build Badge](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiRW4zZUhmeHlJbHRVQnNBZGZEbVJUa0pOK0J0MmtnNVB2dVZZSWhLbUtaNWYxNG96WWg4emN1SjJKL3VSUk9obFl0MnBtajBxejlVWDFiR3ppZGd3U1lrPSIsIml2UGFyYW1ldGVyU3BlYyI6IkFsUkpiMDRkRjZQb1U3Ly8iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master) | ![Build Badge](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiRW4zZUhmeHlJbHRVQnNBZGZEbVJUa0pOK0J0MmtnNVB2dVZZSWhLbUtaNWYxNG96WWg4emN1SjJKL3VSUk9obFl0MnBtajBxejlVWDFiR3ppZGd3U1lrPSIsIml2UGFyYW1ldGVyU3BlYyI6IkFsUkpiMDRkRjZQb1U3Ly8iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=develop) |
```
Comment (WillChilds-Klein) [inline README.md:20]: does the branch name change break these?

Comment (amirhosv) [reply README.md:20]: Fixed it.

**Review by geedo0 — APPROVED** [review]

**Review by WillChilds-Klein — APPROVED** [review]

**Pull request merged**